src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -54,6 +55,15 @@ abstract class By_Day_View extends View {
 	 */
 	protected $grid_days_found_cache = [];
 
+	/**
+	 * An array of cached events for the grid.
+	 *
+	 * @since 5.0.0
+	 *
+	 * @var array
+	 */
+	protected $grid_events = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 0d7eaef0a7779a46ba8de075d11b3fac</sub>